### DISCUSSION TITLE
Running Outline via Docker

### DISCUSSION THREAD

**Discussion by itskunalsingh** [outline/outline#3070] [Ideas]
How can i setup and run outline in Docker on a windows operating system.

**Comment by sebastian-ruiz** [comment]
Take a look here:

https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/doc/documentation-nipGaCRBDu

and:

https://github.com/chsasank/outline-wiki-docker-compose

**Reply by itskunalsingh** [reply]
i have tried but it does not work on windows OS

**Reply by itskunalsingh** [reply]
Using the .env.sample file as a reference add the configuration variables to your environment for your public url, Postgres database, Redis etc. Can you help me with this?

**Comment by gmesml** [comment]


**Reply by tommoor** [reply]
I don't really know what you mean

**Reply by tommoor** [reply]
How is that different from the recommended deployment setup?

**Reply by tommoor** [reply]
Because you haven't provided any of the required environment variables
